### PR TITLE
Create Report 

### PR DESCRIPTION
**Pull request by rzhade3**
Added a new method to client to be able to create a new report

### CODE REVIEW

File: fixtures/vcr_cassettes/create_report.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      - DYNAMIC
+      Server:
+      - cloudflare
+      Cf-Ray:
+      - 58f654d83ea1bc9c-SEA
+      Cf-Request-Id:
+      - '028dd35b1e0000bc9c75365200000001'
+    body:
+      encoding: ASCII-8BIT
+      string: !binary |-
```
[the remark this answers is not in the document]
Comment (rzhade3) [reply fixtures/vcr_cassettes/create_report.yml:77]: I didn't even look at that file 🤔 Looks like whenever you create a report, it dumps this Base64 encoded blob containing the report, reporter and program. The test program we're using has a super long policy which made this encoded string super long, I can re- record this with a shorter policy.

Also, is there a way to auto decode this string?

[the remark this answers is not in the document]
Comment (rzhade3) [reply fixtures/vcr_cassettes/create_report.yml:77]: Similar to this: https://github.com/oreoshake/hackerone-client/blob/master/fixtures/vcr_cassettes/report.yml

[the remark this answers is not in the document]
Comment (oreoshake) [reply fixtures/vcr_cassettes/create_report.yml:77]: Not sure 🤷‍♂️ did you try re-recording? The documentation says a JSON blob should be returned. If it's still the same after a re-record, I'll just merge this and move on.

[the remark this answers is not in the document]
Comment (rzhade3) [reply fixtures/vcr_cassettes/create_report.yml:77]: Tried re- running it a couple times with the same result 🤷 It is indeed returning a report, just in a base64 encoded string. I can make the tests a little bit more stringent but it works rn

**Comment by oreoshake** [comment]
@rzhade3 this is the last PR I'll accept without signed commits 😆

**Pull request merged**